server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/AntlrVariableDefinitionNode.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.model.tree;
+
+
+import org.eclipse.lsp.cobol.core.CobolParser;
+import org.eclipse.lsp.cobol.core.visitor.VariableDefinitionDelegate;
+
+/**
+ * The class represents a variable definition.
+ * This store ANTLR context to be used in {@link VariableDefinitionDelegate}. This is a temporal solution and must be
+ * removed.
+ */
+public class AntlrVariableDefinitionNode extends Node {
+  private CobolParser.DataDescriptionEntryFormat1Context format1Context;
```

Comment:
Maybe, it would be enough to have only one field with the super type. I think that dispatching should be able to resolve the type on the overloaded method call in the `VariableDefintionDelegate`

Reply:
If I use one field then I must write some code to check if it is an instance of something, then cast it to some particular class. Because this is runtime information and it's not available in compilation time.